app/src/main/java/org/groundplatform/android/MainActivity.kt

```diff
@@ -58,6 +61,7 @@ class MainActivity : AbstractActivity() {
   @Inject lateinit var activityStreams: ActivityStreams
   @Inject lateinit var viewModelFactory: ViewModelFactory
   @Inject lateinit var userRepository: UserRepository
+  @Inject lateinit var remoteConfig: FirebaseRemoteConfig
```

Comment:
Would it create problems with unit testing since firebase can't be initialized for unit tests? Have you tried running the tests locally?